_operator_reference/armory.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: Armory Config Reference
+order: 1
+---
+
+{:.no_toc}
+* This is a placeholder for an unordered list that will be replaced with ToC. To exclude a header, add {:.no_toc} after it.
+{:toc}
+
+# Parameters
+
+**spec.spinnakerConfig.config.armory**
+
+```yaml
+armory:
+  dinghy:
+    enabled:
+    templateOrg:
+    templateRepo:
+    githubToken:
+    githubEndpoint:
+    stashUsername:
+    stashToken:
+    stashEndpoint:
+    gitlabToken:
+    gitlabEndpoint:
+    dinghyFilename:
+    autoLockPipelines:
+    fiatUser:
+    notifiers:
+      slack:
+        enabled:
+        channel:
+  diagnostics:
+    enabled:
+    uuid:
+    logging:
+      enabled:
+      endpoint:
+  terraform:
+    enabled:
+    git:
+      enabled:
+      accessToken:
+      username:
+  secrets:
+    vault:
+      enabled:
+      url:
+      path:
+      role:
+      authMethod:
+```
+
+## Dinghy parameters
+
+- `enabled`: true or false.
+- `templateOrg`: SCM organization or namespace where application and template repositories are located.
+- `templateRepo`: SCM repository where module templates are located
+- `githubToken`: GitHub token.
+- `githubEndpoint`: (Default: https://api.github.com) Github API endpoint. Useful if you’re using Github Enterprise.
+- `stashUsername`: Stash username.
+- `stashToken`: Stash token.
+- `stashEndpoint`: Stash API endpoint.
+- `gitlabToken`: GitLab token.
+- `gitlabEndpoint`: GitLab endpoint.
+- `dinghyFilename`: (Default: dinghyfile) Name of the file in application repositories which contains pipelines.
+- `autoLockPipelines`: (Default: true) Lock pipelines in the UI before overwriting on change.
+- `fiatUser`: Fiat user to use for Dinghy operations.
+- `notifiers`:
+  - `slack`:
+    - `enabled`: true or false.
+    - `channel`: Name of channel to send notifications to.
+
+## Diagnostics parameters
+
+- `enabled`: true or false.
+- `uuid`: UUID of the Armory installation
+- `logging`:
+  - `enabled`: true or false.
+  - `endpoint`: Example: `https://debug.armory.io/v1/logs`
+
+## Terraform parameters
```

Comment:
Terraform Integration parameters. This section of parameters is for Armory's Terraform Integration and Terraformer service, not Hashicorp's Terraform. I think it's important we disambiguate